examples/satellite-aws/README.md

```diff
@@ -0,0 +1,148 @@
+
+## Install
+
+### Terraform
+
+Be sure you have the correct Terraform version ( 0.13 or later), you can choose the binary here:
+- https://releases.hashicorp.com/terraform/
+
+### Terraform provider plugins
+
+Be sure you have the compiled plugins on $HOME/.terraform.d/plugins/
+
+- [terraform-provider-ibm](https://github.com/IBM-Cloud/terraform-provider-ibm)
+
+## Usage
+
+```
+terraform init
+```
+```
+terraform plan
+```
+```
+terraform apply
+```
+```
+terraform destroy
+```
+
+## Example Usage
+``` hcl
+provider "ibm" {
+}
+
+provider "aws" {
+  region     = var.aws_region
+  access_key = var.aws_access_key
+  secret_key = var.aws_secret_key
+}
+
+module "satellite-aws" {
+  source = "github.com/terraform-ibm-modules/terraform-ibm-satellite-aws"
+
+  ibmcloud_api_key           = var.ibmcloud_api_key   #pragma: allowlist secret
+  aws_region                 = var.aws_region
+  aws_access_key             = var.aws_access_key
+  aws_secret_key             = var.aws_secret_key
+  resource_group             = var.resource_group
+  is_location_exist          = var.is_location_exist
+  location                   = var.location
+  managed_from               = var.managed_from
+  location_zones             = var.location_zones
+  location_bucket            = var.location_bucket
+  host_labels                = var.host_labels
+  host_provider              = "aws"
+  create_cluster             = var.create_cluster
+  cluster                    = var.cluster
+  cluster_host_labels        = var.cluster_host_labels
+  create_cluster_worker_pool = var.create_cluster_worker_pool
+  worker_pool_name           = var.worker_pool_name
+  worker_pool_host_labels    = var.worker_pool_host_labels
+  create_timeout             = var.create_timeout
+  update_timeout             = var.update_timeout
+  delete_timeout             = var.delete_timeout
+}
+```
+
+## Note
+
+* `satellite-location` module creates new location or use existing location ID/name to process. If user pass the location which is already exist,   satellite-location module will error out and exit the module. In such cases user has to set `is_location_exist` value to true. So that module will use existing location for processing.
+* `satellite-location` module download attach host script to the $HOME directory and appends respective permissions to the script.
+* `satellite-location` module will update the attach host script and will be used in the `user_data` attribute of EC2 module.
+
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Inputs
+
+| Name                                  | Description                                                       | Type     | Default | Required |
+|---------------------------------------|-------------------------------------------------------------------|----------|---------|----------|
+| aws_access_key                        | AWS access key                                                    | string   | n/a     | yes      |
```

Comment:
Where have we defined argument ibmcloud_api_key

Reply:
fixed